app/models/portfolio_item.rb

```diff
@@ -26,8 +26,8 @@ def item_workflow_ref
 
   def validate_workflow
     Approval.call_workflow_api { |api| api.show_workflow(workflow_ref) }
-  rescue Catalog::ApprovalError
-    errors.add(:workflow, "invalid")
-    throw :abort
+  rescue Catalog::ApprovalError => e
+    e.add(422, "Invalid Workflow Reference") if e.status == 404
```

Comment:
@lindgrenj6 This is changing to 400 because the common gem converts it to a 400

Reply:
Ok I'll get that updated.